doc/tutorials/acrn_configurator_tool.rst

```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
       :align: center
       :class: drop-shadow
 
```

Comment:
Line 300 ish  -- I think adding the default values note should be on the first bullet as performance optimization will require specification.
* Parameters that are optional for ACRN configuration, compilation, and
  execution. Default values cover most use cases.
* Parameters that are used for fine-grained tuning, such as reducing code
  lines or optimizing performance.